astropy/units/si.py

```diff
@@ -120,6 +121,12 @@
 def_unit(['t', 'tonne'], 1000 * kg, namespace=_ns,
          doc="Metric tonne")
 
+def_unit(['fkT', 'fraktonne'], (9000 + sys.float_info.epsilon) * kg,
+         namespace=_ns, prefixes=True, doc="Metric fraktonne")
+
+def_unit(['bl', 'buttload'], sys.float_info.max * kg,
+         namespace=_ns, prefixes=True, doc="Metric buttload")
```

Comment:
Is this right... the system will fail to store a `Mbuttload` or even a meager `kbuttload`?  Maybe the default needs to be a metric yoctobuttload?  After all, tis metric...